src/controls/control_manager.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef CONTROL_MANAGER_HPP
+#define CONTROL_MANAGER_HPP
+
+#include "../comms/rm_CAN.hpp"
+
+/// @brief enums for each control type
+enum control_types {
+    PID = 0,
+    // ... the rest of the control types
+};
+
+class control_manager {
```

Comment:
Rename this to ControlManager to follow style guide

Reply:
Also, doxygen is throwing undocumentation warnings at you for control_manager btw